core/settings/date.py

```diff
@@ -60,6 +60,9 @@
         ["fa-linkedin-in", "https://www.linkedin.com/company/datateknologerna-vid-%C3%A5bo-akademi-rf/"],
         ["fa-github", "https://github.com/Datateknologerna-vid-Abo-Akademi"],
     ],
+    "ASSOCIATION_LOGO": "core/images/albin.ico",
+    "ASSOCIATION_HEADER_LOGO": "core/images/albin_black.png",
+    "ASSOCIATION_FOOTER_LOGO": "core/images/albin_black.png",

```

Comment:
Could just have a static/*/core/headerlogo.png etc. that is overridden for each site, would rather not add stuff to settings if it's possible to avoid